openassessment/templates/openassessmentblock/student_training/student_training_unavailable.html

```diff
@@ -7,12 +7,11 @@
 {% endblock %}
 
 {% block button %}
-    <button class="ui-slidable" aria-expanded="false" disabled>
+    <button class="ui-slidable" aria-expanded="false" disabled></button>
```

Comment:
Don't close the button here-- it's closed elsewhere (perhaps in another included template). This is probably the cause of the button not rendering properly.